src/flicking.js

```diff
@@ -775,6 +779,41 @@ eg.module("flicking", ["jQuery", eg, window, document, eg.MovableCoord], functio
 		},
 
 		/**
+		 * Set container's height according to each children's height
+		 * @param direction
+     */
+		_setAdaptiveHeight: function(direction) {
+			var MC = eg.MovableCoord;
+			var $panel;
+			var $first;
+			var $children;
+			var height;
+
+			if (direction === MC.DIRECTION_LEFT) {
+				$panel = this.getNextElement();
+			} else if (direction === MC.DIRECTION_RIGHT) {
+				$panel = this.getPrevElement();
+			} else {
+				$panel = this.getElement();
+			}
+
+			$first = $(":first-child", $panel);
+			height = $first.data("height");

```

Comment:
jQuery's `.data()` store data using internal cache data object.
https://github.com/jquery/jquery/blob/3.0.0/src/data/Data.js#L15

Is ok, but to not be complicated just put the value as element's attribute
```js
height = $first[0].height;
```